operator/config/gke-addon/image_configmap.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 data:
-  cnrm.controller: gcr.io/gke-release/cnrm/controller:7a86865
-  cnrm.deletiondefender: gcr.io/gke-release/cnrm/deletiondefender:7a86865
-  cnrm.recorder: gcr.io/gke-release/cnrm/recorder:7a86865
-  cnrm.unmanageddetector: gcr.io/gke-release/cnrm/unmanageddetector:7a86865
-  cnrm.webhook: gcr.io/gke-release/cnrm/webhook:7a86865
+  cnrm.controller: gcr.io/gke-release/cnrm/cnrm/controller:2fa0f72
+  cnrm.deletiondefender: gcr.io/gke-release/cnrm/cnrm/deletiondefender:2fa0f72
+  cnrm.recorder: gcr.io/gke-release/cnrm/cnrm/recorder:2fa0f72
+  cnrm.unmanageddetector: gcr.io/gke-release/cnrm/cnrm/unmanageddetector:2fa0f72
+  cnrm.webhook: gcr.io/gke-release/cnrm/cnrm/webhook:2fa0f72
```

Comment:
This is skeptical, may due to recent changes to some release scripts. But I don't think it should block the 1.125 release. I'll investigate offline

Reply:
I checked the first one. The image exists for `gcr.io/gke-release/cnrm/controller:2fa0f72`. There is no image for `gcr.io/gke-release/cnrm/cnrm/controller:2fa0f72`.

Reply:
Could we manually update the image path for these?

I believe the plan is to push for a Config Connector version upgrade in the GKE addon to this version, as it resolves the GKE node pool issue which is fixed in this release.

/cc @cheftako

Reply:
Good catch. We should be generating these images at the new paths. Can we have @xiaoweim look at why the image(s) was not generated?

Reply:
Searched around, this is the only change we had around the image in the past month https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2667#discussion_r1773743059
Though I don't see how that relates to this change. @xiaoweim This currently blocks 1.125 release. Could you investigate?

Reply:
cc @justinsb just FYI.

Reply:
Sorry I should have said why we have a redundant "cnrm/cnrm"